oauth2client/client.py

```diff
@@ -811,13 +811,13 @@ def _do_refresh_request(self, http_request):
       raise AccessTokenRefreshError(error_msg)
 
   def _revoke(self, http_request):
-    """Revokes the refresh_token and deletes the store if available.
+    """Revokes this token and deletes the stored copy (if available).
```

Comment:
well, i don't think we have a context where a given `OAuth2Credentials` object would ever be mapped to more than one refresh token, right?

but in any event, i'm happy to tweak.